lib/geordi/cucumber.rb

```diff
@@ -108,6 +111,15 @@ def command_line_options
         # last arg. Do it manually here.
         last_arg = argv.last
         args << last_arg if (last_arg && last_arg.start_with?('-'))
+
+        args.map do |option|
+          if option.match?(/\S\s\S/)
+            # put options with a whitespace in quotes. Relevant for negated tag options ('not @tag')
+            option.prepend('"').concat('"')
```

Comment:
Instead of duplicating the Aruba step, couldn't you wrap the Option with single quotes here?

Reply:
Yes I thought about that. I simply preferred double quotes in the output and did not want to make compromises because of testing. But if you like single quotes just as much or think it's not worth to duplicate the Aruba step for this, I'll change it.

Reply:
IMHO choosing ' or " is rather cosmetic, while duplication of a complex piece of code from a library is WET. If you don't mind, please change as suggested.